tests/utils/deployment/deployment.go

```diff
@@ -235,6 +235,10 @@ func RedefineWithPriviledgedContainer(deployment *v1.Deployment) *v1.Deployment
 	return deployment
 }
 
+func RedefineWithHostPid(deployment *v1.Deployment, hostIpc bool) {
```

Comment:
in the receiving args -> hostIpc = hostPid ?

Reply:
Ah, copy/paste mistake, good catch :+1: